### ISSUE TITLE
Example errors during R CMD Check

### ISSUE THREAD

**Issue by pat-s** [mlr-org/styler.mlr#33]
### Description

in `rules-line-break.R`.

@lorenzwalthert Not sure if these examples are important for users, shall we just remove them?

(If you know how to quickly fix it this is also nice, but if such example errors might eventually come up again in the future we can also remove them? I don't want to bother you with such meaningless things :) )

### Reproducible example

```r
  > ## Don't show: 
  > if (rlang::is_installed("data.tree")) (if (getRversion() >= "3.4") withAutoprint else force)({ # examplesIf
  + ## End(Don't show)
  + # identical
  + styler:::create_tree("x <- 2")
  + styler:::create_tree("c(x <- 2)")
  + ## Don't show: 
  + }) # examplesIf
  > styler:::create_tree("x <- 2")
  Error in text_to_flat_pd(text, transformers, more_specs = more_specs) : 
    argument "more_specs" is missing, with no default
  Calls: <Anonymous> ... %>% -> file.exists -> <Anonymous> -> cache_make_key
  Execution halted
```

**Comment by lorenzwalthert** [comment]
Yeah, delete it.

**Comment by pat-s** [comment]
fixed in 911f19b5071767ca4747b054b6d3eac6fab563be
